core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -168,8 +170,17 @@ oppia.factory('trainingModalService', [
                   editorContextService.getActiveStateName();
                 var state = explorationStatesService.getState(currentStateName);
 
+                // Retrieve the interaction ID.
+                var interactionId = stateInteractionIdService.savedMemento;
+
+                var rulesServiceName = interactionId.charAt(0).toLowerCase() +
+                interactionId.slice(1) + 'RulesService';
```

Comment:
@seanlip I have tried to put this one line of code into a service but it looks messy. It's too weird to name a service that returns the name of some other service. Almost all the services (by convention) end with "..Service" and something like interactionRulesServiceNameService doesn't make sense. Is it fine if we deviate a little and do not make this one service end with "Service"?